css/src/components/messages.scss

```diff
@@ -0,0 +1,58 @@
+.messages {
+	display: flex;
+	flex-direction: column;
+	max-width: 100%;
+	gap: $layout-4;
```

Comment:
`.messages` is only a flex container with a few atomics applied. I don't think we need it. 

```html
<div class="display-flex flex-direction-column gap-md">

Reply:
If images become necessary, it _might_ be a good idea to keep this container around but switch it to a more proscriptive grid

Reply:
removed messages class